docs/react/toggling-devices.mdx

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 6
+---
+
+# Toggling Devices
+
+Fishjam Cloud provides functionality to control media device streams more dynamically. This includes both soft control, such as muting, and hard control, like physically turning the device on or off.
+
+## Soft Mode - Muting and Unmuting
```

Comment:
**suggestion (blocking):** Inconsistent capitalization. We are capitalizing only the first word in any heading.

Reply:
d